src/ext/BalExtension/mba/core/core.csproj

```diff
@@ -14,6 +14,7 @@
     <CreateDocumentationFile>true</CreateDocumentationFile>
     <CodeAnalysisTreatWarningsAsErrors>true</CodeAnalysisTreatWarningsAsErrors>
     <FxCopEnabled>true</FxCopEnabled>
+    <SignOutput Condition="'$(PleaseSignOutput)'!=''">true</SignOutput>
```

Comment:
Can this be centralized to the Tools\WixBuild.csproj.props/targets?

Reply:
This is the only managed DLL that may be redistributed. Given the perf hit of Authenticode signing managed assemblies, do you want them all to be signed? In that case, should be we signing all EXEs and DLLs under \src as well as all MSIs and not just the bundle? Per the PR description, I was intending to keep this change to just redistributed binaries. I imagine not too many build machines could successfully use Device Guard given the very heterogenous build systems in play these days.

Reply:
Ahh, that makes sense. The other managed DLLs that get redistributed are DTF, in a managed custom action. We probably need to sign those as well then.

It is weird to spread `PleaseSignOutput` around the code (it was meant to be hidden away in the infrastructure) so maybe for managed code projects we want signed, we should add a property like `Signable` or `ShouldSign` (or some better word than I'm coming up with right now) and move setting the `SignOutput` property to the infrastructure. That would allow `PleaseSignOutput` to be hidden again...